doc/vim-go.txt

```diff
@@ -1454,6 +1458,13 @@ the `hello_world.go` file is used.
 >
   let g:go_template_file = "hello_world.go"
 <
+                                                     *'g:go_template_no_file'*
+
+Specifies that, rather than using a template, the directory name is used as the
+package name if a new Go file is created. Checkout |'g:go_template_autocreate'| for more info. By default it is disabled.
```

Comment:
Should be `.. the package name is used if a new Go file is created`

Also can you wrap this just like the other paragraphs.